### DISCUSSION TITLE
Using weblate with localization texts in relational DB (PostgreSQL)

### DISCUSSION THREAD

**Discussion by braniq** [WeblateOrg/weblate#10274] [Q&A]
Hello,

do have an idea how to integrate localization texts which I have in form of SQL statement in my Spring boot project to Weblate?
I mean I have standard INSERT INTO... statement where I insert localization texts to DB table. But I want to manage this texts via Weblate. During deployment of the service SQL statements are executed by Flyway. The service itself is providing API and need persistent storage, Postgres in this case, therefore I don't use locales in .properties etc. The problem is Weblate has no direct support for localization in SQL files.

Thank you.
B.

**Comment by nijel** [comment]
Maybe use CSV instead which is supported by Weblate and load that using `COPY` SQL statement?
